src/supermarket/Gemfile.lock

```diff
@@ -862,4 +863,4 @@ DEPENDENCIES
   yard
 
 BUNDLED WITH
-   2.1.4
+   2.2.15
```

Comment:
Please run a buildkite full Adhoc build before changing the bundler version.

Reply:
ok i will run the adhoc build once